data/theme.txt

```diff
@@ -1 +1 @@
-dark
\ No newline at end of file
+light
```

Comment:
You became the thing you swore to destroy!

Reply:
There's some sort of dark mode problem, haven't put the time into it yet

This is supposed to be gitignored too :O